utils/connectContract.js

```diff
@@ -18,7 +18,6 @@ function connectContract() {
             contractABI,
             signer
           ); // instantiating new connection to the contract
-          console.log("rsvpContract", rsvpContract);
         } else {
           console.log("Ethereum object doesn't exist!");
```

Comment:
what do you think about throwing a custom error here? `throw new Error('Please connect to the Mumbai network. Instructions here: <link to our curriculum?>')`

Reply:
I think that would be helpful!